source/user/eph.rst

```diff
@@ -14,9 +14,9 @@ Traditionally, vibronic coupling is evaluated with complex mathematical machiner
 However, if one relaxes the BO approximation and assumes that the electrons experience the moving nuclei as a perturbation of the potential, the full Hamiltonian can be approximated by a Taylor expansion with the adiabatic states at equilibrium and the quantized vibrational modes.
 This approach is widely used in solid state calculations for modeling electron/phonon transport properties (see :cite:`giustino2017eph` for more discussions).
 
-PySCF supports first order eph matrix calculation by either Coupled Perturbed Kohn-Sham/Hartree-Fock (CPKS/CPHF) or finite difference. In periodic system, only FFTDF-based Gamma-point calculation is supported with finite difference. In both cases, a geometrically relaxed structure is required, or the structure instability could lead to imaginary vibrational modes.
+PySCF supports the calculation of the first-order electron-phonon (e-ph) coupling matrix by either coupled perturbed Kohn-Sham/Hartree-Fock (CPKS/CPHF) or finite differences. In periodic systems, only FFTDF-based Gamma-point calculations are supported with finite differences. In both cases, a geometrically relaxed structure is required or the structural instability could lead to imaginary vibrational frequencies.
```

Comment:
periodic systems: sentence is not really clear. I imagine the correct interpretation is that 1) only finite differences are implemented, and 2) that they only can be used for Gamma-point calculations.

Last sentence: "structural instability" is not really an apt expression; the point is that the structure is not a local minimum

Reply:
Are you sure this is not undoing a change in master, I'll have a look

Reply:
@verena-neufeld, I'll have to check about this, I know there were two commits over the weekend, I thought they were small changes, but maybe I looked too quickly.

Reply:
Ya I don't _think_ it's reverting anything